src/main/java/backend/Season.java

```diff
@@ -0,0 +1,87 @@
+package backend;
+
+import com.google.gson.Gson;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Scanner;
+
+public class Season {
+    private int currentRound;
+
+    /**
+     * Create a new season.
+     *
+     */
+    public Season() {
+        this.currentRound = 0;
+        // Yin, je kan hier verder alles toevoegen,
+        // ik heb deze class alleen aangemaakt voor de json parser.
+    }
+
+    /**
+     * Gets the current round.
+     *
+     * @return Current round
+     */
+    public int getCurrentRound() {
+        return currentRound;
+    }
+
+    /**
+     * Converts the current Season class to a json object.
+     *
+     * @return String containing a json representation of the current season.
+     */
+    public String toJson() {
+        Gson gson = new Gson();
+        return gson.toJson(this);
+    }
+
+    /**
+     * Converts the jsonString to a Season class.
+     *
+     * @param jsonString A json String representing a Season class
+     * @return A season class
+     */
+    public static Season fromJson(String jsonString) {
```

Comment:
Maybe add a check here to check if the jsonString is really a json string and not just some other string